src/components/ChartSize.jsx

```diff
@@ -10,12 +10,16 @@ const ChartSize = ({
     zoomIn,
     zoomOut,
 }) => (
-    <div className="ciq-menu cq-chart-size">
-        <div className="cq-menu-btn">
-            <ZoomOutIcon className="ic-icon-with-sub cq-zoom-out" tooltip-title={t.translate('Zoom out')} onClick={zoomOut} />
+    <div className="cq-chart-size-menu">
```

Comment:
why do you need to create a new CSS class `cq-chart-size-menu`?

Reply:
@brucebinary, as you see in the designs, buttons of chart size, have separate hover effect which conflict with class `ciq-menu`, of course, I can override it, but think it's better to define a container for these two buttons.

Reply:
@shayanbinary what separate hover effect? They look the same to me.

Reply:
@brucebinary return it back to what it was and handle it by CSS